runtime/verified_gc/gc.c

```diff
@@ -323,7 +297,9 @@ void Impl_GC7_mark_and_sweep_GC1_aux(uint8_t *g, uint64_t *st, uint64_t *st_top,
 // Handwritten
 void darken_root(value root, value *root_ptr) {
   assert(root == *root_ptr);
-  if (Is_block(root) && Wosize_val(root) > 0 && root >= hs && root < he) {
+  struct HeapRange hr = get_heap_range();
```

Comment:
NIT but Can we just save this in hs and he like before(we will save some function calls)

Reply:
I'm not sure if it would matter in benchmarking, but sure. No issues.